index.bs

```diff
@@ -1734,9 +1734,8 @@ during credential generation.
 
 <div dfn-type="enum-value" dfn-for="AttestationConveyancePreference">
     *   <dfn>none</dfn> - indicates that the [=[RP]=] is not interested in [=authenticator=] [=attestation=].
-        The client MAY replace the [=AAGUID=] and [=attestation statement=] generated
-        by the authenticator with meaningless client-generated values. For example, in order to avoid having to obtain
-        [=user consent=] to relay uniquely identifying information to the [=[RP]=], or to save a roundtrip to a Privacy CA.
+        For example, in order to avoid having to obtain [=user consent=] to relay uniquely identifying information to
+        the [=[RP]=], or to save a roundtrip to a Privacy CA.
```

Comment:
Do we need to clarify here that "avoid having to obtain user consent" does not mean that user consent is not required for a makeCredential operation using "none" attestation?

Reply:
I think that's a good point and have included a change in this PR to address this. Although it seems that browsers are unlikely to prompt for consent when no attestation is requested, this spec doesn't forbid that and we shouldn't implicitly suggest otherwise.

Reply:
Ah yes, maybe I'm confusing myself on what entity makes the prompt. What I meant to say is that _authenticators_ are specified to always require consent in authenticatorMakeCredential (regardless of the RP's attestation conveyance preference), but I guess that issue is orthogonal to whether the _browser_ prompts for consent to convey the attestation. Anyway, your change looks good to me. :)